imkar/integrate/integrate.py

```diff
@@ -0,0 +1,62 @@
+import warnings
+from scipy.integrate import trapezoid
+import numpy as np
+
+
+def surface_sphere(data, coords):
+    r"""Integrate over a set of points sampled on a spherical surface.
+
+    .. math::
+
+        S = \int \int data(\phi, \theta)  sin(\theta) d\phi d\theta
+
+    Parameters
+    ----------
+    data : FrequencyData
+        Input data to integrate. Its cshape need to be (..., #theta, #phi)
+    coords : Coordinates
+        Coordinate points at which the data is sampled. Its cshape needs to be
+        (#theta, #phi), matching the cshape of `data`.
+
+    Returns
+    -------
+    FrequencyData
+        The integration result. Its dimension is reduced by the last two,
+        which are consumed by the integration.
+
+    Examples
+    --------
+    >>> from imkar import integrate
+    >>> from pyfar import FrequencyData, Coordinates
+    >>> import numpy as np
+    >>> phi, theta = np.meshgrid(
+    >>>     np.linspace(0, 2*np.pi, num=361),
+    >>>     np.linspace(0, np.pi, num=181))
+    >>> coords = Coordinates(
+    >>>     phi, theta, np.ones(phi.shape), 'sph')
```

Comment:
What about using the [pyfar.samplings](https://pyfar.readthedocs.io/en/latest/modules/pyfar.samplings.html#) module here?

Reply:
it is not really suitable, since we need coords in (#theta, #phi) shape and it is easier to do it by ourself.